### PR TITLE
[UPDATING THE RULE!] Comply with LF/AI Community Charter

### PR DESCRIPTION
**Pull request by myungjoo**
As attorneys of stakeholders are drafting the community charter,
based on the current draft, I am updating related documents.

This is at the early stage of drafting. We have a lot more to update.

Refer to  https://wiki.lfai.foundation/display/NNSTREAM/NNStreamer+CONTRIBUTING+draft+of+2020.04

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -2,22 +2,23 @@
...
 
-## Reviewer
+## Committer (previously Reviewer)
 
-- May reject a pull-request, which prohibits merging.
+- May reject a pull-request, which prohibits merging. "Veto"
 - May vote for an approval.
-- May merge a pull-request if it has enough number of approval-votes from reviewers or maintainers (2 or more).
-- The ability (vote/reject/merge) may be limited to a few subdirectories.
+- (Requires merge privliege) May merge a pull-request if it has enough number of approval-votes from reviewers or maintainers (2 or more).
```
Comment (zhoonit) [inline MAINTAINERS.md:12]: I think this is a typo. privliege -> privilege.

Comment (myungjoo) [reply MAINTAINERS.md]: Fixed. Thanks!

File: MAINTAINERS.md [hunk 2]
```diff
@@ -2,22 +2,23 @@
...
 
-## Reviewer
+## Committer (previously Reviewer)
 
-- May reject a pull-request, which prohibits merging.
+- May reject a pull-request, which prohibits merging. "Veto"
 - May vote for an approval.
-- May merge a pull-request if it has enough number of approval-votes from reviewers or maintainers (2 or more).
-- The ability (vote/reject/merge) may be limited to a few subdirectories.
+- (Requires merge privilege) May merge a pull-request if it has enough number of approval-votes from reviewers or maintainers (2 or more).
```
Comment (anyj0527) [inline MAINTAINERS.md:12]:
```
... from reviewers or maintainers (2 or more).
```
IMHO, ```reviewers``` -> ```committers``` seems appropriate.

Comment (myungjoo) [reply MAINTAINERS.md]: Updated.

**Comment by myungjoo** [comment]
PTAL: @anyj0527 @zhoonit @leemgs @again4you @wooksong @kparichay @jijoongmoon @jaeyun-jung 

I'd like to have this reviewed by most of the committers before the first TSG meeting.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,8 +1,109 @@
...
...n available committer is a committer who has been submitting code commits or code reviews to the main project (nnstreamer.git) during the last 30 days. A senior committer is a committer who has committed codes to the main GitHub project (nnstreamer.git) before other available committers. Unavailability of chairman and deputy can be declared via the mailing-list by a committer. The unavailability declaration becomes effective after the declaring committer tries to contact the chairman or deputy personally, the declaration is acknowledged by other two or more committers, and there are no responses from chairman or deputy via the mailing list or GitHub issues of the main project (nnstreamer.git) within 30 days. With the effective unavailability declaration, the available senior committer becomes the deputy who holds the TSC meeting to elect a chairperson within the next 30 days. If the available senior committer fails to do this within the given 30 days after the effective declaration, the Linux Foundation Series manager may designate any TSC voting member of committer as a deputy chairperson and any TSC voting member or committer may contact Linux Foundation to initiate this process.
```
Comment (zhoonit) [inline CONTRIBUTING.md:45]: Q. I am having hard time to catch the meaning of  `A senior committer`. Would the definition include a person who hasn't been committed for a long time but committed `before other available committers`?

Comment (myungjoo) [reply CONTRIBUTING.md:45]: Do you suggest that "the most senior among available committers" will be more clear than "the most senior and available committer"?

Comment (zhoonit) [reply CONTRIBUTING.md:45]: I think that is more clear. Thank you for the clarification 👍

Comment (myungjoo) [reply CONTRIBUTING.md:45]: Updated!

**Comment by myungjoo** [comment]
@jijoongmoon @leemgs @wooksong @again4you @jaeyun-jung @kparichay 
LF AI will announce NNStreamer at 9am PT, 4/16. Please review before it.

**Comment by myungjoo** [comment]
Let's confirm this at the next TSC. Merging it due to no objections so far.

**Pull request merged**